.github/stale.yml

```diff
@@ -0,0 +1,42 @@
+# Stale config for https://github.com/marketplace/stale
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - security
+  - blocked
+
+# Label to use when marking a pull request as stale
+staleLabel: stale
+
+pulls:
+  # Number of days of inactivity before a pull request becomes stale
+  daysUntilStale: 7
+
+  # Number of days of inactivity before a stale pull request is closed
+  daysUntilClose: 7
```

Comment:
This gives 2 weeks in total. Should we bump it slightly higher? 2-week vacations are not uncommon (especially during winter holidays).

Reply:
sure, but we can always increase these values if they end up wrong

also: if PRs get closed while on vacay, it's easy enough to re-open them